frontend/src/components/Board/Settings/index.tsx

```diff
@@ -317,17 +372,114 @@ const BoardSettings = ({ isOpen, setIsOpen, socketId }: Props) => {
 																: board.maxVotes
 														}
 													/>
-												)}
+												</ConfigurationSettings>
 											</>
 										)}
 									</Flex>
 								</StyledAccordionContent>
 							</StyledAccordionItem>
-						</Accordion>
 
+							{isSubBoard && (isStakeholderOrAdmin || isOwner || isSAdmin) && (
+								<StyledAccordionItem value="responsible">
+									<StyledAccordionHeader>
+										<StyledAccordionTrigger>
+											<Text heading="5">Team Responsible</Text>
+											<StyledAccordionIcon name="arrow-down" />
+										</StyledAccordionTrigger>
+									</StyledAccordionHeader>
+									<StyledAccordionContent>
+										<Flex direction="column" gap={16}>
+											<ConfigurationSettings
+												text="Change responsible participant for this board."
+												title="Team Responsible"
+												isChecked={isUsersChecked}
+												handleCheckedChange={handleUsers}
+											>
+												<Flex align="center" css={{ mt: '$10' }}>
+													<Text
+														css={{
+															mr: '$8',
+															color: '$primary300'
+														}}
+													>
+														Responsible Lottery
+													</Text>
+													<Separator
+														orientation="vertical"
+														css={{
+															'&[data-orientation=vertical]': {
+																height: '$12',
+																width: 1
+															}
+														}}
+													/>
+
+													<Flex
+														css={{
+															height: '$24',
+															width: '$24',
+															borderRadius: '$round',
+															border: '1px solid $colors$primary400',
+															ml: '$12',
+															cursor: isUsersChecked
+																? 'pointer'
+																: 'default',
+
+															transition: 'all 0.2s ease-in-out',
+
+															'&:hover': isUsersChecked
+																? {
+																		backgroundColor:
+																			'$primary400',
+																		color: 'white'
+																  }
+																: 'none'
+														}}
+														align="center"
+														justify="center"
+														onClick={handleLottery}
+													>
+														<Icon
+															name="wand"
+															css={{
+																width: '$12',
+																height: '$12'
+															}}
+														/>
+													</Flex>
+
+													<Text
+														size="sm"
+														color="primary800"
+														css={{ mx: '$8' }}
+													>
+														{responsible?.firstName}{' '}
+														{responsible?.lastName}
+													</Text>
+
+													<Avatar
+														css={{ position: 'relative' }}
+														size={32}
+														colors={{
+															bg: highlight2Colors.highlight2Lighter,
+															fontColor:
+																highlight2Colors.highlight2Dark
+														}}
```

Comment:
You try use without import the color and use it like stitches way?

Something like:
```suggestion
														colors={{
															bg: '$highlight2Lighter',
															fontColor: '$highlight2Dark'
'														}}
```

Reply:
Already tested it and works without import `highlight2Colors`.